raw_cloud_filter:
    apply_filter: true
    filter_boat_points: true
    filter_range: true
    filter_intensity: true
    debug_cloud: true

    boat_length: 0.8 # meters
    boat_width: 2.4 # meters
    max_height: 0.6 # meters
    max_xy_range: 50.0 # meters
    min_intensity: 10

pointcloud2scan:
    angle_resolution: 2.0 # degrees 

livox2scan:
    input_frame: "livox_frame"
    output_frame: "base_link"
    x: 0.0
    y: 0.0
    z: 0.0
    roll: 180.0 # degrees
    pitch: 0.0 # degrees
    yaw: 0.0 # degrees

ping2scan:
    input_frame: "ping_frame"
    output_frame: "base_link"
    x: 0.0
    y: 0.0
    z: 0.0
    roll: 0.0
    pitch: 0.0
    yaw: 0.0
